statsmodels/discrete/discrete_model.py

```diff
@@ -3163,6 +3191,23 @@ def _estimate_dispersion(self, mu, resid, df_resid=None):
         a = ((resid**2 / mu - 1) * mu**(-q)).sum() / df_resid
         return a
 
+    def _get_start_params(self, **kwargs):
```

Comment:
Really similar to the version above; in the future there may be room for de-duplication